README.md

```diff
@@ -9,3 +9,17 @@
 - [Theo]()
```

Comment:
Can you put the collaborator's section at the bottom, and give you section a heading `## CI/CD & Jenkins`

Can you also separate the sections with lines? `---`